### PR TITLE
fix: Missing test of view field count

### PR DESCRIPTION
**Pull request by energinet-ajw**
The test didn't catch three missing is_tax fields in the schema contracts. The problem was the test assertion.
This also means that the documentation doesn't have an is_tax field description. The assertion does catch if the actual field count is less than the field count in the contract.

## Pull-request quality

- [x] The title adheres to [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [ ] Tests are written and executed locally
- [ ] Subsystem tests have been tested (by manually deploying to `dev_002`)

### CODE REVIEW

File: source/databricks/calculation_engine/contracts/data_products/settlement_report/monthly_amounts_v1.py [hunk 1]
```diff
@@ -42,5 +42,8 @@
         #
         # EIC or GLN number
         t.StructField("charge_owner_id", t.StringType(), nullable),
+        #
+        # Taxation type
```
Comment (BjarkeMeier) [inline source/databricks/calculation_engine/contracts/data_products/settlement_report/monthly_amounts_v1.py:46]: Type?

File: source/databricks/calculation_engine/package/common/schemas.py [hunk 2]
```diff
@@ -84,6 +84,13 @@ def assert_schema(
             Actual field names: {actual.fieldNames()}."""
         )
 
+    if len(actual_fields) > len(expected_fields):
```
Comment (BjarkeMeier) [inline source/databricks/calculation_engine/package/common/schemas.py:87]: Seems like we're missing a test of this function

File: source/databricks/calculation_engine/contracts/data_products/settlement_report/monthly_amounts_v1.py (hunk 1 above)
Comment (energinet-ajw) [reply source/databricks/calculation_engine/contracts/data_products/settlement_report/monthly_amounts_v1.py:46]: Removed type.

File: source/databricks/calculation_engine/package/common/schemas.py (hunk 2 above)
Comment (energinet-ajw) [reply source/databricks/calculation_engine/package/common/schemas.py:87]: Test added.

**Review by BjarkeMeier — APPROVED** [review]

**Pull request merged**